keyboards/hs60/v1/rules.mk

```diff
@@ -2,3 +2,5 @@
 # when we get USB suspend event. We want it to keep updating
 # backlight effects.
 NO_SUSPEND_POWER_DOWN = yes
+
+DEFAULT_FOLDER = hs60/v1/iso
```

Comment:
Could we not introduce this, and move/duplicate this file to each revision.

Reply:
This is for backward compatibility, until the `DEFAULT_FOLDER` stuff is actually removed.

Reply:
We dont necessarily need to care about backwards compatibility, given this is targeting develop.

And adding in more `DEFAULT_FOLDER`s, just makes the overall removal process more painful.

Reply:
Fair enough. Done.